libs/designsystem/src/lib/components/dropdown/dropdown.component.ts

```diff
@@ -44,7 +44,8 @@ import { KeyboardHandlerService } from './keyboard-handler.service';
   ],
 })
 export class DropdownComponent
-  implements AfterContentChecked, AfterViewInit, OnDestroy, ControlValueAccessor {
+  implements AfterContentChecked, AfterViewInit, OnDestroy, ControlValueAccessor
+{
```

Comment:
Is this correct?

Reply:
This is a prettier-specific configuration related to line-breaks and/or line length, so I think we should leave it as is.